### DISCUSSION TITLE
Make "Nature Instances" optionally NonSelectable

### DISCUSSION THREAD

**Discussion by valentinwinkelmann** [visualdesigncafe/nature-renderer#89] [Ideas]
The Nature Instance feature for prefabs is fantastic, I love it, but can we please get an option to make the objects non-selectable? Since they always get a big bounding box which is used for selection it can block all other objects. For example, if i have trees and now click in the scene i keep accidentally selecting a nature instance.

If I also group them (e.g.: all trees in a “Tree” group and mark this group as non-selectable (the finger symbol next to the eye symbol in the hierarchy), then clicking on such a Nature Instance object does not result in it no longer being selectable but in a Unity Editor freeze that appears in “Hold on (Busy for mm:ss)...
Application.Mesage.LeftButtonUp
Waiting for Unity's code to finish executing.”, then it only helps to crash unity with the task manager and restart. Maybe just add a checkbox under “Window->Nature Renderer->[X]Selectable Nature Instances”. that would really help.

**Comment by maxplooi** [comment]
Thank you for letting me know about this. I have updated the behavior so that nature instances that are not pickable in the scene view are properly ignored. This fixes the crash as well. The fix is included in version **6.0.1**. You can download the latest version via the Asset Store, please give it a try.
